pages/post/[postId].js

```diff
@@ -55,6 +53,7 @@ export default function Post({ post, isDynamicLink }) {
     const router = useRouter()
     useEffect(() => {
         if (isDynamicLink) router.push(`/post/${post.id}`, undefined, { shallow: true })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [isDynamicLink])
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Disabling eslint rules should be avoided as much as possible because they are there for a reason. In this case, the rule `react-hooks/exhaustive-deps` is disabled which ensures that all dependencies of the useEffect hook are specified in the dependency array. If you're sure about your implementation and want to ignore the warning, please add a comment explaining why you're disabling this rule.
```diff
-    // eslint-disable-next-line react-hooks/exhaustive-deps
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    // Disabling this rule because we only want to run the effect when `isDynamicLink` changes
     }, [isDynamicLink])
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->